cristin-project/src/main/java/no/unit/nva/cristin/projects/CristinApiClient.java

```diff
@@ -121,29 +125,28 @@ public SearchResponse queryCristinProjectsIntoWrapperObjectWithAdditionalMetadat
     }
 
     protected HttpResponse<String> queryProjects(Map<String, String> parameters, QueryType queryType)
-        throws ApiGatewayException {
+            throws ApiGatewayException {
 
         URI uri = attempt(() -> generateQueryProjectsUrl(parameters, queryType))
-            .toOptional(failure ->
-                logError(ERROR_MESSAGE_QUERY_WITH_PARAMS_FAILED, queryParameters(parameters), failure.getException()))
-            .orElseThrow();
+                .toOptional(failure ->
+                        logError(ERROR_MESSAGE_QUERY_WITH_PARAMS_FAILED,
+                                queryParameters(parameters), failure.getException()))
+                .orElseThrow();
```

Comment:
You are removing the stack trace from the thrown exception. attempt(()->....).orElseThrow(fail->logAndThrow())
Be careful that the logAndThrow method **returns** an exception that orElseThrow will throw.

Reply:
I think this is some technical debt I have created earlier on...